migrations/3_deploy_relay.js

```diff
@@ -5,8 +5,8 @@ module.exports = function(deployer, network, accounts) {
   // https://etherscan.io/token/0x9e46a38f5daabe8683e10793b06749eef7d733d1#readContract totalSupply
   const TOTAL_SUPPLY = '1885913075851542181982426285';
 
-  // https://coinmarketcap.com/currencies/polyswarm/ retrieved on 5/28/18
-  const NCT_ETH_EXCHANGE_RATE = 80972;
+  // https://coinmarketcap.com/currencies/polyswarm/ retrieved on Dec 31st 2018
+  const NCT_ETH_EXCHANGE_RATE = 15300;
```

Comment:
NCT_ETH_EXCHANGE_RATE = amount of NCT equivalent to 1 ETH

Current ETH/NCT exchange rate is 0.00001506 ETH/NCT (just fetched from coinmarketcap)

1.0 / 0.00001506 ~= 66401 NCT/ETH

Reply:
thanks, i'm dumb